example/06-local-volumesnapshotlocation.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 the Heptio Ark contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: velero.io/v1
+kind: VolumeSnapshotLocation
+metadata:
+  name: <LOCATION_NAME>
+  namespace: velero
+spec:
+  provider: openebs.io/cstor-blockstore
+  config:
+    namespace: <OPENEBS_NAMESPACE>
```

Comment:
it is ok to repeat the comment here on what the namespace means.

Reply:
ok